packages/bvaughn-architecture-demo/components/console/ConsoleRoot.tsx

```diff
@@ -42,8 +42,12 @@ export default function ConsoleRoot({
             messageListRef={messageListRef}
             showSearchInputByDefault={showSearchInputByDefault}
           >
-            <div className={styles.ConsoleRoot} data-test-id="ConsoleRoot">
-              <div className={styles.TopRow}>
+            <div
+              className={styles.ConsoleRoot}
+              data-menu-open={isMenuOpen}
```

Comment:
I'm curious why you used a data attribute here rather than a different `className`? It's not a big deal, but the latter would be more consistent with how the other styles work in the package.

Reply:
Mostly because it felt awkward to use BEM style syntax in CSS modules, but moved it to a simple class name so things are consistent 😎